components/navbar/sidebar.snip.html

```diff
@@ -17,11 +17,14 @@
 }}
 
 <!-- Start Sidebar -->
-<amp-sidebar id="{{sidebar-id}}" class="ampstart-sidebar px3 {{^sidebaronly}}md-hide lg-hide{{/sidebaronly}}" layout="nodisplay">
+<amp-sidebar id="{{sidebar-id}}" class="ampstart-sidebar px3 {{^sidebaronly}}md-hide lg-hide{{/sidebaronly}} {{sidebarclasses}}" layout="nodisplay">
```

Comment:
I am going to write a design doc to come up with an idea to separate markup , layout and spacing so all of this is easily configurable. Since i saw a lot of PRs adding classes i thought it might be easy to segregate this was, because my design will take a while to be ready for implementation. However this is not very imp at this point.